os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/trace_uart.c

```diff
@@ -215,23 +206,23 @@ bool trace_uart_tx(uint8_t *pstr,uint16_t len, UART_TX_CB tx_cb)
         int ret;
         UART_TXDMAConfig(TRACE_UART_DEV, 8);
         UART_TXDMACmd(TRACE_UART_DEV, ENABLE);
-        ret = UART_TXGDMA_Init(TRACE_UART_INDEX, &p_uart_obj->UARTTxGdmaInitStruct,(void *) p_uart_obj, traceuart_dma_tx_complete,pstr,len);
+        ret = UART_TXGDMA_Init(TRACE_UART_INDEX, &p_uart_obj->UARTTxGdmaInitStruct,(void *) p_uart_obj, traceuart_dma_tx_complete, pstr, len);
         NVIC_SetPriority(GDMA_GetIrqNum(0, p_uart_obj->UARTTxGdmaInitStruct.GDMA_ChNum), TRACEUART_DMA_PRIO);
 
         if (!ret ) {
-            platform_debug("%s Error(%d)\n", __FUNCTION__, ret);
+            dbg("Error(%d)\r\n", ret);
```

Comment:
`dbg` is not always printed.
How about defining its own dbg macro based on ERROR, WARN, INFO level?
You can refer to `https://github.com/Samsung/Tiz`enRT/blob/master/os/include/debug.h`.

Reply:
I agree with Jeongchan's opinion about separating message level as ERROR, INFO.
You can define them as bledbg and blevdbg. Because dbg is working even we don't want to see ble logs when we enable CONFIG_DEBUG and CONFIG_DEBUG_ERROR. And if we don't split the logs as error and information, then it shows too much logs.

Reply:
We will raise another PR to fix all debug log and printf.
All the debug log will be review and change accordingly.
Current commit is just to fix the main issue.

Reply:
ok